### PR TITLE
Switch to self hosted runners

### PR DESCRIPTION
**Pull request by cweagans**
## Description

This needs tested somehow.

Ref: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/7776

Ref: https://github.com/department-of-veterans-affairs/va.gov-team/issues/36310

Ref: https://github.com/department-of-veterans-affairs/va.gov-team/issues/36192

## Testing done

https://github.com/department-of-veterans-affairs/content-build/pull/949#issuecomment-1028403668

## Definition of done
- [ ] content-release moved to self-hosted runner

### CODE REVIEW

**Review by ndouglas — APPROVED** [review]

**Review by oseasmoran73 — APPROVED** [review]

**Comment by cweagans** [comment]
Verified that requesting the self hosted runner works as expected in https://github.com/department-of-veterans-affairs/content-build/actions/runs/1786439837

**Review by mydesignrocks — APPROVED** [review]
LGTM

File: .github/workflows/content-release.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ env:
 
 jobs:
   start-runner:
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, linux, x64]
```
Comment (indytechcook) [inline .github/workflows/content-release.yml:32]: This Job is what started the runner: https://github.com/department-of-veterans-affairs/content-build/pull/949/files#diff-90571dbe8d8f4805db215f91f9d177538a3043d2d07cf1fcd29199bee34cfa79L69  

Can someone help me understand how this works?

Comment (indytechcook) [reply .github/workflows/content-release.yml:32]: Answering my own question:  the `[self-hosted, linux, x64]` is more generic and the `Start EC2 Runner` starts a larger more dedicated instance which is used in the build step with `runs-on: ${{ needs.start-runner.outputs.label }}`

**Pull request merged**